frontend/src/App.js

```diff
@@ -3,13 +3,35 @@ import Header from './Header/Header';
 import Footer from './Footer/Footer';
 import BulletinBoard from './Pages/Bulletin/BulletinBoard';
 import DisqusComments from './Pages/Comments/Comments';
+<<<<<<< HEAD
+import FormFeedback from './Pages/FormFeedback/FormFeedback';
+import NewBulletinBoard from './Pages/NewBulletinBoard/BulletinBoard';
+import Beneficiaries from './Pages/NewBeneficiaries/Beneficiaries';
+import Grid from './Pages/CommentCard/CommentGrid';
+=======
 import Beneficiaries from './Pages/Beneficiary/Beneficiaries';
 import CommentPage from './Pages/CommentCard/CommentPage';
 import About from './Pages/About/About';
+>>>>>>> 448bea360cd1b0be2381b7befa4400154fa891f3
 import './App.css';
 
 function App() {
   return (
+<<<<<<< HEAD
```

Comment:
<<<< HEAD 
such lines should not be there. It is an indication that code has not been merged properly.